docker/centos8/Dockerfile

```diff
@@ -355,6 +355,17 @@ RUN mkdir -p /opt/foundationdb/old && \
 
 RUN curl -Ls https://github.com/manticoresoftware/manticoresearch/raw/master/misc/junit/ctest2junit.xsl -o /opt/ctest2junit.xsl
 
+# Download Rust binaries
```

Comment:
Please take a look at how the `golang` install is done above. The sticky bit is the sha256sum of the binary file that is being downloaded. We use that as a protection against downloading an incorrect/corrupt binary. The team at SF is pretty particular about this pattern, it is driven by their infosec policies.